bql/semantic/expression_test.go

```diff
@@ -195,6 +189,22 @@ func TestBooleanEvaluationNode(t *testing.T) {
 	}
 }
 
+func buildLiteralOrDie(textLiteral string) *literal.Literal {
+	lit, err := literal.DefaultBuilder().Parse(textLiteral)
+	if err != nil {
+		panic(fmt.Sprintf("Could not parse text literal got err: %v", err.Error()))
+	}
+	return lit
+}
+
+func newNodeFromStringOrDie(nodeType, nodeID string) *node.Node {
+	n, err := node.NewNodeFromStrings(nodeType, nodeID)
+	if err != nil {
+		panic(fmt.Sprintf("Could not build node from type %q and value %q", nodeType, nodeID))
+	}
+	return n
+}
+
 func TestNewEvaluator(t *testing.T) {
```

Comment:
The test struct you probably want something called `wantErr` rather than `err` to indicate whether you want an error or not (boolean field).

Reply:
I will be using the next PR to separate the tests that expect an error from the tests that do not. I will put them in two different functions (for both `expression_test.go` and `planner_test.go`). Thank you again for the advice Tati.